examples/v1beta1/github-owners/README.md

```diff
@@ -0,0 +1,41 @@
+## GitHub EventListener
+
+Creates an EventListener that listens for GitHub webhook events.
```

Comment:
Could you give bit more info about this example like what it is and why its used

Reply:
Also specify Github Owner wherever its required in README in order to differentiate between Github and Github Owner